flow-typed/npm/array-move_vx.x.x.js

```diff
@@ -0,0 +1,33 @@
+// flow-typed signature: 15027ec23a7de4cf3f20e202339bcdd2
```

Comment:
These files need to be checked in, in order to provide a repeatable developer experience. Definitions can change for the same version as people improve the libdef. See the package.json for how these are updated.